src/main/java/org/zalando/nakadi/domain/Timeline.java

```diff
@@ -168,4 +172,22 @@ public boolean equals(final Object o) {
     public int hashCode() {
         return id != null ? id.hashCode() : 0;
     }
+
+    public static Timeline createTimeline(
+            final String eventType,
+            final int order,
+            final Storage storage,
+            final String topic,
+            final Date createdAt) {
+        final Timeline timeline = new Timeline(eventType, order, storage, topic, createdAt);
+        timeline.setId(UUID.randomUUID());
+        return timeline;
+    }
+
+    public static Timeline createFakeTimeline(final EventType eventType, final Storage storage) {
```

Comment:
I suppose you meant the fake timeline. This is for ability to switch back from the first real timeline we create in case something goes wrong, because fake timeline won't be created in DB.